chapters/08-mapping.qmd

```diff
@@ -7,3 +7,177 @@ project:
+    GO.reproject(tanzania, GI.crs(tanzania), GFT.EPSG(32736)), 
+    50000), 
+  GFT.EPSG(32736),
+  GFT.EPSG(4326)
+)
+tanzania_neigh = filter!(
+  :geom => x -> GO.intersects(x, only(tanzania_buf.geom)), 
+  GeoDataFrames.read("data/world.gpkg")
+)
+```
+
+## Introduction
+
+A satisfying and important aspect of geographic research is communicating the results.
+Map making---the art of cartography---is an ancient skill that involves communication, intuition, and an element of creativity.
+In addition to being fun and creative, cartography also has important practical applications.
+A carefully crafted map can be the best way of communicating the results of your work, but poorly designed maps can leave a bad impression.
+Common design issues include poor placement, size and readability of text and careless selection of colors, as outlined in the style guide of the Journal of Maps.
+Furthermore, poor map making can hinder the communication of results [@brewer_designing_2015]:
+
+> Amateur-looking maps can undermine your audience's ability to understand important information and weaken the presentation of a professional data investigation.
+
+Maps have been used for several thousand years for a wide variety of purposes.
+Historic examples include maps of buildings and land ownership in the Old Babylonian dynasty more than 3000 years ago and Ptolemy's world map in his masterpiece Geography nearly 2000 years ago [@talbert_ancient_2014].
+
+Map making has historically been an activity undertaken only by, or on behalf of, the elite.
+This has changed with the emergence of open source mapping software such as mapping packages in Python, R, and other languages, and the "print composer" in QGIS, which enable anyone to make high-quality maps, enabling "citizen science".
+Maps are also often the best way to present the findings of geocomputational research in a way that is accessible.
+Map making is therefore a critical part of geocomputation and its emphasis not only on describing, but also changing the world.
+
+Basic static display of vector layers in Julia can be done with the `plot` function from **Makie.jl**, as we saw in Sections @sec-vector-layers and @sec-using-rasterio.
+Other, more advanced uses of these methods, were also encountered in subsequent chapters, when demonstrating the various outputs we got.
+In this chapter, we provide a comprehensive summary of the most useful workflows of these two methods for creating static maps (@sec-static-maps).
+Static maps can be easily shared and viewed (whether digitally or in print), however they can only convey as much information as a static image can.
+Interactive maps provide much more flexibilty in terms of user experience and amount of information, however they often require more work to design and effectively share.
+Thus, in @sec-interactive-maps, we move on to elaborate on the `.explore` method for creating interactive maps, which was also briefly introduced earlier in @sec-vector-layers.
```

Comment:
We may not need to split static/interactive into sections as there's little difference